docker/postgres/entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
They are but they need to be in the docker build context in order to be copied to the docker image and each node has it's own directory. We could do something with a directory structure like
`docker/signed/entrypoint.sh`
`docker/signed/postgres/Dockerfile`
`docker/signed/npm/Dockerfile`
Then we can run docker build for each of the images from the `signed` directory referencing each Dockerfile directly
`docker build -f npm/Dockerfile -t jenkins-build-npm .`
`docker build -f postgres/Dockerfile -t jenkins-build-npm .`

You could also dispense with the signed directory altogether and put the entrypoint.sh script in the `docker` directory but I've tried to keep that for build files for Jenkins itself. 

I don't  mind which way we do it. The above with a new top level folder is a bit more confusing in my mind but we're going to automate this stuff soon anyway so it doesn't matter greatly. 
Let me know if you want me to set it up this way.

Reply:
Ah, I didn't know about the Docker context stuff, thanks.

I think I agree that the new folder structure would be confusing. If we did the second option, does that change anything about how we build the images? Would you have to make sure that you run `docker build` from the `docker` directory rather than `docker/npm`, etc.?

Reply:
If we move the entrypoint.sh script into a parent directory like the `docker` directory, you won't be able to run the docker build from inside the original directory because it will be missing the file to copy. The only way you can build the wrong thing is if you do something like `docker build -f npm/Dockerfile -t jenkins-build-plugin-updates .` and reference the wrong file but to be fair, that's possible now if you get the tag name wrong. 
So you will have to build the images differently but hopefully we'll be automating these image builds soon so it shouldn't matter greatly.

Reply:
Let's do that, then. Could you update the [Readme section on node images](https://github.com/nationalarchives/tdr-jenkins#deploy-jenkins-node-images) to say which folder it should be run from?

Reply:
Ok, I've created a `signed` directory within the `docker` directory and put the entrypoint.sh script there and the images in their own folders. I've updated the README with how to build them.

Reply:
Sorry, sorry, I think we might have been talking at cross purposes. I thought we both agreed that putting them into a `signed` directory would be confusing! 🙂 

I thought you were just going to move the entrypoint script to the docker directory? Sorry for the confusion!

Reply:
There's already an `entrypoint.sh` in the docker directory that's used for Jenkins itself. So we would have to start calling it `entrypoint-for-signed-commits.sh` or something similar in that directory. It is confusing having a sub directory but I don't think it's as bad as mixing the files for Jenkins itself with the files for its nodes.

Reply:
I've put all of the node image directories into a nodes directory. The shared entrypoint script is in there and now there is only one docker command for all images. I've updated the README to say this.

Reply:
That's great, thanks!